tests/TestFiles/Strings/Strings1.hs

```diff
@@ -17,7 +17,8 @@ capABC (x:xs) = x:capABC xs
 capABC "" = ""
 
 appendEq :: String -> String
-appendEq s = s ++ "!"
+appendEq s@(_:_) = s ++ "!"
+appendEq _ = "..."
```

Comment:
Avoid modifying existing tests (unless there is some good justification here?), but we can add this as a new test appendEq2 if there is a reason to.